rust/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+newline_style = "Native"
```

Comment:
what are the other styles?

Reply:
Possible values: "Auto", "Native", "Unix", "Windows"

Reference: https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=#newline_style